frontend/src/components/Sidebar/Sidebar.jsx

```
@@ -1,18 +1,420 @@
+      <div
+        className="flex items-center px-4 py-4 border-b border-[#222] relative"
+        ref={workspaceMenuRef}
+      >
+        <div className="flex items-center px-4 py-4 border-b border-[#222] relative">
+          <div className="w-9 h-9 bg-green-500 rounded-full flex items-center justify-center text-white font-bold text-lg">
+            W
+          </div>
+          <span
+            className="ml-3 text-white font-semibold truncate cursor-pointer"
+            onClick={() => setWorkspaceMenuOpen((v) => !v)}
+          >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)